### ISSUE TITLE
Subconference Call for Papers configuration has just one welcome text field but offers to switch language on CfP page

### ISSUE THREAD

**Issue by Duckattack** [frab/frab#431]
On the CfP page (https://frab.froscon.de/en/deRSE18/cfp/) I can switch between DE and EN via a slider, but there is no possibility in the CfP configuration to give the text in both languages.

**Comment by manno** [comment]
These buttons toggle the interface language. 

Note: we have a (problematic) multi language input form for the notifications, which could be re-used, improved on.

**Comment by manno** [comment]
I agree it's an UX issue. But we're lacking the database models, the admin interfaces and the views.

Like I said above, we already failed at a similar problem with the [notifications editor](https://github.com/frab/frab/blob/master/app/views/conferences/edit_notifications.html.haml). 

Actually looking at it again, it's not as complicated as notifications, more like [rooms](https://github.com/frab/frab/blob/master/app/views/conferences/_form_rooms.html.haml) or [links](https://github.com/frab/frab/blob/master/app/views/events/_form.html.haml#L39).

PRs are welcome.
